src/dsymbol.d

```diff
@@ -166,12 +166,16 @@ alias PASSinline = PASS.PASSinline;
 alias PASSinlinedone = PASS.PASSinlinedone;
 alias PASSobj = PASS.PASSobj;
 
+// Search options
 enum : int
 {
     IgnoreNone              = 0x00, // default
     IgnorePrivateMembers    = 0x01, // don't find private members
     IgnoreErrors            = 0x02, // don't give error messages
     IgnoreAmbiguous         = 0x04, // return NULL if ambiguous
+    IgnoreImports           = 0x08, // don't find symbols in imports
+    IgnoreThisModule        = 0x10, // only look in imports
+    IgnoreRoot              = 0x20, // this search is for a root module
```

Comment:
So make another PR with just the enum rename refactoring and rebase this one on top after it's pulled.  We do that all the time and it works just fine.